Documentation/upgrades/upgrade_3_0.md

```diff
@@ -116,4 +116,21 @@ $ ETCDCTL_API=3 etcdctl endpoint health
 127.0.0.1:22379 is healthy: successfully committed proposal: took = 18.513301ms
 ```
 
+## Further considerations
+
+- etcdctl environment variables have been updated. If you find that
+    `ETCDCTL_API=2 etcdctl cluster-health` works properly but `ETCDCTL_API=3
+    etcdctl endpoints health` responds with `Error:  grpc: timed out when
+    dialing`, make sure you are using the [new variable
+    names](https://github.com/coreos/etcd/tree/master/etcdctl#etcdctl).
+
+## Known Issues
+
+- etcd 3.0.x does not work properly on Go 1.7+. See [Issue
```

Comment:
Let me know if the changes I just pushed help clarify the problem and solutions a bit better. :-)